sycl/plugins/cuda/pi_cuda.hpp

```diff
@@ -21,8 +21,7 @@
 #define _PI_CUDA_PLUGIN_VERSION 1
```

Comment:
I suggest spelling the rules how/when this version is changing.
My proposal would be to increment it by 1 with every change to the plugin.

Reply:
Do you think this should be described in `pi_cuda.hpp` (and in every other plugin that follows this convention), or does it belong in `pi.h`?

Reply:
I think this can be suggested at pi.h level but clearly we cannot require it there.
May add a suggestion there, and a more strict guidance in each plugin.
It'd be great aid if we could somehow automate this

Reply:
Change made. I have also added _PI_*_PLUGIN_VERSION to all plugins

Reply:
Thanks!